express-api/src/routes/projectRouter.ts

```diff
@@ -0,0 +1,40 @@
+import controllers from '@/controllers';
+import express from 'express';
+
+const router = express.Router();
+
+export const PROJECT_DISPOSAL = '/projects/disposal';
+export const PROJECT_REPORTS = '/projects/reports';
+
+//These originally had a separate route for numeric id and projectNumber, but I don't think express supports this pattern.
+router.route(`${PROJECT_DISPOSAL}/:projectId`).get(controllers.getDisposalProject);
+router.route(`${PROJECT_DISPOSAL}/:projectId`).put(controllers.updateDisposalProject);
+router.route(`${PROJECT_DISPOSAL}/:projectId`).delete(controllers.deleteDisposalProject);
```

Comment:
```suggestion
router.route(`${PROJECT_DISPOSAL}/:projectId`)
  .get(controllers.getDisposalProject)
  .put(controllers.updateDisposalProject)
  .delete(controllers.deleteDisposalProject);
```
If the route is the same, you can always chain the methods together like this. Might be easier to read.